installer/alpine/init-x86_64

```diff
@@ -542,7 +644,7 @@ fi
 if [ -f "$ovl" ]; then
 	ebegin "Loading user settings from $ovl"
 	# create apk db and needed /dev/null and /tmp first
-	apk add --root $sysroot --initdb --quiet --no-progress
+	apk add --root $sysroot --initdb --quiet
```

Comment:
why remove `--no-progress`? will apk to the right thing of not showing the progress when run in CI?